seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java

```diff
@@ -65,6 +75,8 @@ public class RestApiIT {
 
     private static final String HOST = "http://localhost:";
 
+    private static final Integer PORT = 8080;
```

Comment:
we can use `node1Config.getEngineConfig().getHttpConfig().getPort()` to get port, if hardcode here and config updated, the test will be fail.